src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java

```diff
@@ -115,10 +113,7 @@ private OutputConsumerPath(Path destination, boolean isJar, boolean keepFsOpen,
 			Map<String, String> env = new HashMap<>();
 			env.put("create", "true");
 
-			FileSystem fs = FileSystems.newFileSystem(uri, env);
-			if (fs.isReadOnly()) throw new IOException("the jar file "+destination+" can't be written");
-
-			destination = fs.getPath("/");
+			destination = FileSystems.newFileSystem(uri, env).getPath("/");
```

Comment:
Why? This was specifically added to detect a common setup problem early. Bad rebase?

Reply:
yea I think so